client/signup/processing-screen/index.jsx

```diff
@@ -309,7 +310,11 @@ export class SignupProcessingScreen extends Component {
 	}
 
 	renderActionButton() {
-		const { loginHandler, translate } = this.props;
+		const { flowName, loginHandler, translate } = this.props;
+
+		if ( isAutocontinueFlow( flowName ) ) {
```

Comment:
@jancavan -- should we not show the disabled button to **any** flow that declares itself to be "auto continue" or should we limit to `import` for now?

Reply:
Are there any repercussions in doing it for any flow? I know for the regular Signup Flow, we had some discussion around doing this, but it was not (yet) implemented. I think for now, let's just limit it to WISF.